Clinical-Genomics/cg
# Adds tests for cli/set/sample

This PR tests the cli set sample. N.B! the testing skips the lims-dependency when setting gender of the sample, please make sure this works on stage as described below!

How to test:
0. install cg on stage on rasta
1. us
1. run 'cg set sample [sample_id] -s [sex]
1. open stage-lims

Expected outcome:
The gender propagated to LIMS

also. Build successful
number of tests increased
coverage increased
